pkg/cloudprovider/types.go

```diff
@@ -63,6 +74,9 @@ type CloudProvider interface {
 	// IsDrifted returns whether a NodeClaim has drifted from the provisioning requirements
 	// it is tied to.
 	IsDrifted(context.Context, *v1.NodeClaim) (DriftReason, error)
+	// RepairPolicy is for CloudProviders to define a set Unhealthy condition for Karpenter
+	// to monitor on the node.
+	RepairPolicy() []RepairStatement
```

Comment:
Any reason you can't do this? 
```suggestion
	RepairPolicies() []RepairPolicy
```